app/scripts/metamask-controller.test.js

```diff
@@ -3982,6 +3987,77 @@ describe('MetaMaskController', () => {
           'This Secret Recovery Phrase has already been imported.',
         );
       });
+
+      ///: BEGIN:ONLY_INCLUDE_IF(multi-srp)
```

Comment:
For future reviewers: The `multi-srp` build feature has been removed, but code-fencing is not used in tests, so that shouldn't break anything

Though, we should probably remove/change it at some point.